specification/security/resource-manager/readme.md

```diff
@@ -472,9 +467,9 @@ input-file:
 - Microsoft.Security/stable/2021-06-01/assessments.json
 - Microsoft.Security/stable/2022-01-01/alerts.json
 - Microsoft.Security/stable/2022-05-01/settings.json
-- Microsoft.Security/stable/2023-01-01/pricings.json
```

Comment:
Why is this path removed from here in readme?

Reply:
It got replaced with the 2024-01-01 stable version